README.md

```diff
@@ -9,6 +9,20 @@ https://github.com/protocolbuffers/protobuf/releases/tag/v3.20.0
 ```bash
 go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.27.1
 ```
+## proto file destination
+
+Those 2 file are used to generate go binding
+- `fig/tendermint/codec/v1/codec.proto`
+- `fig/tendermint/transform/v1/transform.proto`
```

Comment:
I think there could be more than 2 – see [this](https://github.com/streamingfast/proto-ethereum/tree/develop/sf/ethereum). Our Makefile should take that into account.

Reply:
Only documenting what we currently have. The Makefile is likely to change a lot once we work on automation of proto file generation.

Reply:
Fair enough 👍